java/java-guestbook/.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+// extensions.json will trigger an installation prompt for the Cloud Code 
```

Comment:
nor any of the .vscode files

Reply:
These are actually samples for the VSCode extension, so this folder is actually the most important part!